src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -50,9 +50,9 @@
     <Reference Include="FontAwesome.WPF">
       <HintPath>..\..\extern\FontAwesome\FontAwesome.WPF.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.5892.25330, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\extern\greg\Greg.dll</HintPath>
+    <Reference Include="Greg, Version=1.0.5950.12824, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.0.5950.12824\lib\net45\Greg.dll</HintPath>
+      <Private>True</Private>
```

Comment:
I think I discussed this with @ikeough  long before.
The project-assigned value of CopyLocal is determined in the following order: 
1.If the reference is another project, called a project-to-project reference, then the value is true.
2.If the assembly is found in the global assembly cache, the value is false.
3.As a special case, the value for the mscorlib.dll reference is false.
4.If the assembly is found in the Framework SDK folder, then the value is false.
5.Otherwise, the value is true.